<?xml version="1.0"?>

<launch>

	<!-- client -->
	<node pkg="provider_client" type="client" name="client">
		<param name="SECURE_PROVIDER_INTERFACE" value="false"/>
		<param name="SECURE_ARROWHEAD_INTERFACE" value="false"/>
		<param name="OR_BASE_URI" value="http://localhost:8440/orchestrator/orchestration"/>
		<param name="OR_BASE_URI_HTTPS" value="https://localhost:8441/orchestrator/orchestration"/>
		<param name="CLIENT_PORT" value="8468"/>
		<param name="CLIENT_ADDRESS" value="130.240.5.68"/>
		<param name="CLIENT_ADDRESS6" value="[fe80::1a57:58d9:c43e:6319]"/>
		<param name="CONSUMER_ID" value="TestconsumerID"/>
		<param name="CLIENT_SYSTEM_NAME" value="client_ros_insecure"/>
		<param name="AUTHENTICATION_INFO" value="null"/>
		<param name="SERVICE_DEFINITION" value="IndoorTemperature_ProviderExample"/>
		<param name="INTERFACES" value="REST-JSON-SENML"/>
		<param name="SECURITY" value="token"/>
		<param name="OVERRIDE_STORE" value="true"/>
		<param name="MATCHMAKING" value="true"/>
		<param name="METADATA_SEARCH" value="false"/>
		<param name="PING_PROVIDERS" value="false"/>
		<param name="ONLY_PREFERRED" value="true"/>
		<param name="EXTERNAL_SERVICE_REQUEST" value="false"/>
		<param name="PROVIDER_SYSTEM_NAME" value="InsecureTemperatureSensor"/>
		<param name="PROVIDER_ADDRESS" value="130.240.5.68"/>
		<param name="PROVIDER_PORT" value="8469"/>
	</node>

</launch>
